packages/cli/package.json

```diff
@@ -89,7 +93,7 @@
     "chai-as-promised": "^7.1.1",
     "chai-bignumber": "^2.0.2",
     "chai-string": "^1.5.0",
-    "ganache-cli": "^6.4.0",
+    "ganache-cli": "^6.3.0",
```

Comment:
Because I have tested 6.3.0 to work properly with ZepKit and pinned ZepKit to that version but didn't test 6.4.0. I thought it wouldn't hurt to stay for everyone on the same version.